app/views/Export/ExportConfirmUpload.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Alert, NativeModules } from 'react-native';
+
+import { Icons } from '../../assets';
+import exitWarningAlert from './exitWarningAlert';
+import ExportTemplate from './ExportTemplate';
+
+const MOCK_ENDPOINT =
+  'https://private-anon-da01e87e46-safeplaces.apiary-mock.com/upload';
```

Comment:
maybe these urls could be a `.env` var?

General question.. when might we change them? would they change per environment/flavour.. or?

If static, leave as is

Reply:
These will be part of the selected HA eventually -- the HA retrieval will need to be a two step process to 
1. query the yaml file
2. query the HA's config endpoint

Once we have the staging server wired with a `cursor.json` file with the configuration meta data, we can wire this up completely

Reply:
Ah, right. makes sense